css-animationworklet/Overview.bs

```diff
@@ -848,96 +843,8 @@ There are no known privacy issues introduced by these features.
 Examples {#examples}
 ====================
 
-Example 1: Hidey Bar. {#example-1}
```

Comment:
With the removal of this should we call out that this is one use case which isn't currently possible?

Reply:
This seems like a job for EXPLAINER which I have partially updated. I plan to add the same example but using "input for worklet" instead.